iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer.hpp

```diff
@@ -36,22 +36,24 @@ class RelativePointer : public BaseRelativePointer
     using const_ptr_t = const T*;
     using ptr_t = T*;
 
+    using BaseRelativePointer::BaseRelativePointer;
```

Comment:
This added code will be acceptable? let me know your comments on this? 'Rule A12-1-6'

Reply:
@saif-at-github do you know if doxygen shows the ctors inherited this way? If yes, then this is totally fine. If not, we need to think how we can make this visible to the user.

cc @MatthiasKillat